<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="EVT01213NB" failures="0" tests="1" name="AppiumPractise.AppiumFrameworkPractise.ApiDemoTest" time="9.151" errors="1" timestamp="2020-04-30T21:43:50 EET" skipped="0">
  <testcase classname="AppiumPractise.AppiumFrameworkPractise.ApiDemoTest" name="ApiDemoApp" time="9.151">
    <error message="Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find &#039;adb.exe&#039; in [&quot;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\platform-tools\\adb.exe&quot;,&quot;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\emulator\\adb.exe&quot;,&quot;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\adb.exe&quot;,&quot;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\bin\\adb.exe&quot;]. Do you have Android Build Tools installed at &#039;C:\Users\z003srtv\AppData\Local\Android\Sdk;C:\Users\z003srtv\AppData\Local\Android\Sdk\platform-tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools\bin;&#039;?
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;EVT01213NB&#039;, ip: &#039;192.168.192.1&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;10.0.2&#039;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find &#039;adb.exe&#039; in [&quot;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\platform-tools\\adb.exe&quot;,&quot;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\emulator\\adb.exe&quot;,&quot;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\adb.exe&quot;,&quot;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\bin\\adb.exe&quot;]. Do you have Android Build Tools installed at &#039;C:\Users\z003srtv\AppData\Local\Android\Sdk;C:\Users\z003srtv\AppData\Local\Android\Sdk\platform-tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools\bin;&#039;?
    at getResponseForW3CError (C:\Users\z003srtv\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\z003srtv\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:392:37)
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;EVT01213NB&#039;, ip: &#039;192.168.192.1&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;10.0.2&#039;
Driver info: driver.version: AndroidDriver" type="org.openqa.selenium.SessionNotCreatedException">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\platform-tools\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\emulator\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\z003srtv\AppData\Local\Android\Sdk;C:\Users\z003srtv\AppData\Local\Android\Sdk\platform-tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools\bin;'?
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'EVT01213NB', ip: '192.168.192.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '10.0.2'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\platform-tools\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\emulator\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\z003srtv\AppData\Local\Android\Sdk;C:\Users\z003srtv\AppData\Local\Android\Sdk\platform-tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools\bin;'?
    at getResponseForW3CError (C:\Users\z003srtv\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\z003srtv\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:392:37)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'EVT01213NB', ip: '192.168.192.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '10.0.2'
Driver info: driver.version: AndroidDriver
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:336)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:88)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:98)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:94)
at AppiumPractise.AppiumFrameworkPractise.Base.Capabilities(Base.java:88)
at AppiumPractise.AppiumFrameworkPractise.ApiDemoTest.ApiDemoApp(ApiDemoTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.reflect.InvocationTargetException
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)
... 43 more
Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\platform-tools\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\emulator\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\z003srtv\AppData\Local\Android\Sdk;C:\Users\z003srtv\AppData\Local\Android\Sdk\platform-tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools\bin;'?
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'EVT01213NB', ip: '192.168.192.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '10.0.2'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\platform-tools\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\emulator\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\adb.exe","C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\platform-tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools;C:\\Users\\z003srtv\\AppData\\Local\\Android\\Sdk\\tools\\bin;\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\z003srtv\AppData\Local\Android\Sdk;C:\Users\z003srtv\AppData\Local\Android\Sdk\platform-tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools;C:\Users\z003srtv\AppData\Local\Android\Sdk\tools\bin;'?
    at getResponseForW3CError (C:\Users\z003srtv\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\z003srtv\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:392:37)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
at java.base/java.util.stream.ReferencePipeline.findFirst(Unknown Source)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
... 48 more
]]>
    </error>
  </testcase> <!-- ApiDemoApp -->
  <system-out/>
</testsuite> <!-- AppiumPractise.AppiumFrameworkPractise.ApiDemoTest -->
